tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
Today I've made very simple filter for tags, urls and digits in the universaldetector.py:

![img1](https://cloud.githubusercontent.com/assets/10864177/6249627/c122366a-b788-11e4-9d3a-a802ffaa45be.png)

Before filtering:
<code>
Ran 413 tests in 97.956s
FAILED (failures=26)
</code>
After filtering:
<code>
Ran 413 tests in 57.866s
FAILED (failures=16)
</code>

Sorry for the picture but this markdown interpreter has a problem to display code above correctly.